ivy_tests/test_ivy/conftest.py

```diff
@@ -2,24 +2,57 @@
 import os
 import pytest
 from typing import Dict
+import sys
+import multiprocessing as mp
+
+
+mod_frontend = {
+    "tensorflow": None,
+    "numpy": None,
+    "jax": None,
+    "torch": None,
+    "mindspore": None,
+    "scipy": None,
+    "paddle": None,
+}  # multiversion
+mod_backend = {
+    "numpy": None,
+    "jax": None,
+    "tensorflow": None,
+    "torch": None,
+    "paddle": None,
+    "mxnet": None,
+}  # multiversion
+
+multiprocessing_flag = False  # multiversion
+
 
 # local
 import ivy_tests.test_ivy.helpers.test_parameter_flags as pf
 from ivy import set_exception_trace_mode
 from ivy_tests.test_ivy.helpers import globals as test_globals
 from ivy_tests.test_ivy.helpers.available_frameworks import available_frameworks  # noqa
+from ivy_tests.test_ivy.helpers.multiprocessing import backend_proc, frontend_proc
 
 GENERAL_CONFIG_DICT = {}
 UNSET_TEST_CONFIG = {"list": [], "flag": []}
 UNSET_TEST_API_CONFIG = {"list": [], "flag": []}
 
 TEST_PARAMS_CONFIG = []
 UNSUPPORTED_FRAEMWORK_DEVICES = {"numpy": ["gpu", "tpu"]}
-
 if "ARRAY_API_TESTS_MODULE" not in os.environ:
     os.environ["ARRAY_API_TESTS_MODULE"] = "ivy.functional.backends.numpy"
 
 
+def default_framework_mapper(fw, set_too=False):
+    # do a path search, get the latest one
+    versions = os.listdir(f"/opt/fw/{fw}")
+    versions.sort()
+    if set_too:
+        sys.path.insert(1, f"/opt/fw/{fw}/{versions[-1]}")
```

Comment:
Ideally, we shouldn't use `/` but instead `os.path` so it can handle different path separators on different OS.